pages/seller_financing/index.tsx

```diff
@@ -0,0 +1,354 @@
+  TextInput,
+  Table,
+  Card,
+  Space,
+  Select,
+  LoadingOverlay,
+} from '@mantine/core'
+import { useRouter } from 'next/router'
+import { IconShoppingCart } from '@tabler/icons'
+import OrderSummary from '../../components/OrderSummary'
+import ErrorAlert from '../../components/ErrorAlert'
+import { formatCurrency, getProducts, getTotals } from '../../utils/products'
+import { CustomerType } from '../../utils/email'
+
+declare global {
+  interface Window {
+    initializeSlope: any
+    Slope: any
+    SlopeJs: any
+  }
+}
+
+const toTermKeyLabel = (termKey: string) => {
+  let key = termKey.replace('_', ' ')
+  key = key.charAt(0).toUpperCase() + key.slice(1)
+  return key
+}
+
+const Checkout: React.FC<{
+  customerForm: Record<string, any>
+  setCustomerForm: any
+}> = ({ customerForm, setCustomerForm }) => {
+  const router = useRouter()
+  const [loading, setLoading] = useState(false)
+  const [error, setError] = useState(null)
+  const [primaryColor, setPrimaryColor] = useState('')
+  const [product, setProduct] = useState('Soda')
+  const products = getProducts(product)
+  const totals = getTotals(products)
+  const [total, setTotal] = useState(totals.total)
+  const [isLegacySDK, setIsLegacySDK] = useState(false)
+  const [customer, setCustomer] = useState<any>(null)
+  const [customerSecret, setCustomerSecret] = useState<string | null>()
+  const [financingTerm, setFinancingTerm] = useState<string | null>(null)
+  const [loadingEstimatedPayout, setLoadingEstimatedPayout] = useState(false)
+  const [estimatedDiscountedPayout, setEstimatedDiscountedPayout] = useState<any[] | null>(null)
+  const selectedDiscountedPayout = estimatedDiscountedPayout?.find(
+    (dp) => dp.termKey === financingTerm
+  )
+
+  const localeSelector = ''
+
+  const onChangeRedirect = (event) => {
+    setCustomerForm({ ...customerForm, mode: event.currentTarget.checked ? 'redirect' : null })
+  }
+  const guestMode = false
+
+  const fetchEstimatedDiscountedPayout = async (
+    customerId: string,
+    secret: string,
+    orderTotal: number
+  ) => {
+    setLoadingEstimatedPayout(true)
+    const resp = await fetch('/api/estimated-discounted-payout', {
+      method: 'POST',
+      headers: {
+        Authorization: secret,
+      },
+      body: JSON.stringify({
+        customerId,
+        total: orderTotal,
+      }),
+    })
+    const updatedEstimatedDiscountedPayout = await resp.json()
+    setEstimatedDiscountedPayout(updatedEstimatedDiscountedPayout.discountedPayouts)
+    setLoadingEstimatedPayout(false)
+  }
+
+  useEffect(() => {
+    if (customer && customerSecret) {
+      fetchEstimatedDiscountedPayout(customer.id, customerSecret, total)
+    }
+  }, [customer?.id, total, financingTerm])
+
+  const initCustomer = async () => {
+    const customerResp = await fetch('/api/create-customer', {
+      method: 'POST',
+      body: JSON.stringify({
+        ...customerForm,
+        email: `test${CustomerType.SKIP_PRE_QUALIFY}@slopepay.com`,
```

Comment:
could we not use the email on the form?

Reply:
I don't want that complexity :\
the form's default is New customer